lnt/server/reporting/analysis.py

```diff
@@ -176,6 +179,12 @@ def get_value_status(self, confidence_interval=2.576,
         elif self.prev_failed:
             return UNCHANGED_PASS
 
+        # Ignore changes if the hash of the binary is the same and the field is
+        # sensitive to the hash, e.g. execution time.
+        if self.ignore_same_hash:
+            if self.cur_hash and self.prev_hash and self.cur_hash == self.prev_hash:
```

Comment:
Not sure about completely ignoring, if same binaries have changes it can be a good indication of the noise level, and changed binaries may also be impacted by the same noise. Not sure if that's possible, but it may be good to display the results for the binaries with same hash separately.